soft4pes/model/grid/rl_grid.py

```diff
@@ -39,41 +43,36 @@ class RLGrid:
         Current state of the grid [p.u.].
     base : base value object
         Base values.
-    data_sim : dict
-        System data.
     """
 
-    def __init__(self, Vgr, fgr, Rg, Lg, base):
+    def __init__(self, Vgr, fgr, Rg, Lg, base, ig_ref_init=None):
+        super().__init__()
         self.Vgr = Vgr / base.V
         self.wg = 2 * np.pi * fgr / base.w
         self.Rg = Rg / base.Z
         self.Xg = Lg / base.L
-        self.x = np.array([0, 0])
         self.base = base
-        self.sim_data = {
-            'x': [],
-            'vg': [],
-            't': [],
-        }
+        self.set_initial_state(ig_ref_init=ig_ref_init)
 
-    def get_discrete_state_space(self, v_dc, Ts):
+    def set_initial_state(self, **kwargs):
         """
-        Get the discrete state-space model of the grid in alpha-beta frame.
-        Discretization is done using the forward Euler method.
+        Set the initial state of the grid based on the grid current reference, if provided.
```

Comment:
What does this mean? I do not get it

Reply:
The user can provide the initial current reference to the system in the initialization.
`    def __init__(self, Vgr, fgr, Rg, Lg, base, ig_ref_init=None):`
The reference is an optional parameter. If user gives it the simulation will start from steady state. 

About the function definition `def set_initial_state(self, **kwargs):` 
**kwargs is used to allow different references as an input, because the machine and the grid need to have the same function definitions. For grid, the current reference is given but for the machine, both torque and flux references are given. 

![image](https://github.com/user-attachments/assets/dff145ee-7a00-47a1-99d7-ac4fd0ba75fb)

Reply:
I think my comment was unclear. I commented on the comment "state of the grid". What does "state of the grid" mean? The grid does not have a "state"